### ISSUE TITLE
Add scrape sentinel info

### ISSUE THREAD

**Issue by liuyk0109** [oliver006/redis_exporter#405]
**What version of redis_exporter are you running?**
[√ ] 0.3x.x
[ ] 1.x.x

Do we have some plans to add scrape sentinel info like:
```
[root@server10 bin]# ./redis-cli -h 192.169.2.52 -p 26379 -i 1 info Sentinel 
# Sentinel
sentinel_masters:3
sentinel_tilt:0
sentinel_running_scripts:0
sentinel_scripts_queue_length:0
sentinel_simulate_failure_flags:0
master0:name=user03,status=sdown,address=192.169.2.52:6381,slaves=1,sentinels=5
master1:name=user02,status=ok,address=192.169.2.54:6380,slaves=1,sentinels=5
master2:name=user01,status=ok,address=192.169.2.52:6379,slaves=1,sentinels=5
```
The `status` is very useful for me. Hope it can be added to the exporter.

**Comment by oliver006** [comment]
Agreed, that would definitely be helpful.
I don't use Redis with sentinel and am very short of time right now but I'll gladly welcome a PR to add this.

**Comment by oliver006** [comment]
Closed by #418 - thx @houstonheat !
